native-gradle-plugin/buildSrc/src/main/groovy/org.graalvm.build.functional-testing.gradle

```diff
@@ -72,27 +72,34 @@ configurations {
 gradlePlugin.testSourceSets(sourceSets.functionalTest)
 configurations.functionalTestImplementation.extendsFrom(configurations.testImplementation)
 
-['functionalTest', 'fullFunctionalTest'].each {
+def graalVm = javaToolchains.launcherFor {
+    languageVersion.set(JavaLanguageVersion.of(11))
```

Comment:
This is used for this build only: it sets up what version of Java we test with. It's not forced on the user. Alternatively we could create a test matrix, but it will significantly increase build time.